packages/embarkjs-connector-web3/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "embarkjs-connector-web3",
   "version": "4.0.0-beta.0",
-  "author": "Jonathan Rainville",
+  "author": "Iuri Matias <iuri.matias@gmail.com>",
```

Comment:
We don't have any sort of generic email address we can use that ties to messages on gitter do we?

Reply:
I don't think so. @iurimatias ?